Exiled.Events/Patches/Events/Player/FlippingCoin.cs

```diff
@@ -39,32 +43,24 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             LocalBuilder ev = generator.DeclareLocal(typeof(FlippingCoinEventArgs));
 
             int offset = 0;
-            int index = newInstructions.FindIndex(instruction => instruction.opcode == OpCodes.Ldc_I4_S) + offset;
-
-            // Extract the old labels, before removing it
-            List<Label> oldLabels = newInstructions[index].ExtractLabels();
-
-            // Remove both NW events and logic
-            newInstructions.RemoveRange(index, 45);
-
-            // Redirect the old labels to the new starting point
-            newInstructions[index].WithLabels(oldLabels);
+            int index = newInstructions.FindIndex(instruction => instruction.LoadsConstant(109)) + offset;
 
             offset = -1;
-            index = newInstructions.FindLastIndex(instruction => instruction.LoadsField(Field(typeof(Coin), nameof(Coin.RateLimiter)))) + offset;
+            int replaseIndex = newInstructions.FindIndex(instruction => instruction.Calls(PropertyGetter(typeof(Random), nameof(Random.value)))) + offset;
```

Comment:
The `switch` at this index doesn't seem to have labels.

Reply:
It is has
<img width="493" alt="image" src="https://user-images.githubusercontent.com/79921583/211041493-75bea2d4-05e4-43a2-918e-49de744bec98.png">
<img width="482" alt="image" src="https://user-images.githubusercontent.com/79921583/211041553-40f97453-26b9-42ef-8b34-e8db5cd92daf.png">

Reply:
Sorry I looked at the operand of br.s and then at the wrong index.